ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -1,6 +1,8 @@
 # Fabric Variables
 
 - The fabric underlay and overlay topology variables, define the elements related to build the L3 Leaf and Spine fabric.
+- It is possible to provision configurations for a complete topology but flag devices as not deployed using the host variable `is_deployed`.
+  - Interfaces on devices that face an undeployed device can also be shutdown using the variable `is_deployed_shutdown_peer_interfaces`. This is also useful when trying to satisfy the `eos_validate_state` interface tests.
```

Comment:
1. extra space at the beginning
2. regarding the `eos_validate_state` - I would be more explicit stating that if this setting is not used AND some Device Under Test (DUT) has some peer devices are using `is_deployed: false`, `eos_validate_state` role will fail on the tests trying to verify that DUT interfaces are up.

Reply:
Agreed

Reply:
I've tried to explain this in it's own section